packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -110,4 +113,29 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
 
         this.rowContainerYOffset = rowContainerYOffset;
     }
+
+    private notifyFocusedCellRecycling(): void {
+        let tableFocusedElement = this.table.shadowRoot!.activeElement;
+        while (
+            tableFocusedElement !== null
+            && !(tableFocusedElement instanceof TableCellView)
+        ) {
+            if (tableFocusedElement.shadowRoot) {
+                tableFocusedElement = tableFocusedElement.shadowRoot.activeElement;
+            } else {
+                break;
+            }
+        }
+        if (tableFocusedElement instanceof TableCellView) {
+            tableFocusedElement.focusedRecycleCallback();
+        }
+        if (this.table.openActionMenuRecordId !== undefined) {
+            const row = Array.prototype.find.call(
+                this.table.rowContainer.children,
```

Comment:
Think it just looks a little cleaner to do `Array.from(this.table.rowContainer.children).find(...)`

Reply:
Made that change in the row code. Here updated to use `children` as you suggested in another comment.